### PR TITLE
Log in

### PR DESCRIPTION
**Pull request by d-r-holden**
Please Review

### CODE REVIEW

File: controllers/sessions.js [hunk 1]
```diff
@@ -12,9 +12,11 @@ const SessionsController = {
     const password = req.body.password;
     User.findOne({ email: email }).then((user) => {
       if (!user) {
-        res.render("user/login", { layout: "login", error: "incorrect email"});
+        res.render("login", { layout: "login", error: "incorrect email"});
```
Comment (ScAziz) [inline controllers/sessions.js:15]: I think we can strip out the layout passed in here.

File: controllers/sessions.js [hunk 2]
```diff
@@ -12,9 +12,11 @@ const SessionsController = {
...
+        console.log("ERROR")
       } else if (user.password != password) {
-        res.render("user/login", {layout: "login", error: "incorrect password"});
+        res.render("login", {layout: "login", error: "incorrect password"});
```
Comment (ScAziz) [inline controllers/sessions.js:18]: Same as above.

File: views/userdashboard.ejs [hunk 3]
```diff
@@ -19,7 +19,7 @@
         <ul>
           <li><a href='/events'>Events</a></li>
           <li><a href='/requests'>My Requests</a></li>
-          <li><a href='/wishlist'>Wishlist</a></li>
+          <li><a href='/wishlists'>Wishlist</a></li>
```
Comment (ScAziz) [inline views/userdashboard.ejs:23]: Nice catch.

File: views/signup.ejs [hunk 4]
```diff
@@ -27,6 +27,10 @@
   
 </form>
 
+<form method = 'GET' action = 'login'>
```
Comment (ScAziz) [inline views/signup.ejs:30]: Is this just to catch to people who've navigated to sign-up by mistake but already have an account?

File: controllers/sessions.js (hunk 1 above)
Comment (d-r-holden) [reply controllers/sessions.js:15]: Agree, I kept it in just incase.

**Review by ScAziz — COMMENTED** [review]
Ciao Ciao, looks clean and some good spots! Just left some comments for tidying up a little , we should make sure its up to date with main without breaking changes as well xx

File: views/signup.ejs (hunk 4 above)
Comment (d-r-holden) [reply views/signup.ejs:30]: It is indeed.

**Review by ScAziz — APPROVED** [review]

**Pull request merged**